numpy/distutils/misc_util.py

```diff
@@ -113,10 +113,9 @@ def quote_args(args):
     # don't used _nt_quote_args as it does not check if
     # args items already have quotes or not.
     args = list(args)
-    for i in range(len(args)):
-        a = args[i]
-        if ' ' in a and a[0] not in '"\'':
-            args[i] = '"%s"' % (a)
+    for idx, arg in enumerate(args):
+        if ' ' in arg and arg[0] not in '"\'':
+            args[idx] = f'"{arg}"'
```

Comment:
I don't see anywhere in numpy or scipy that uses this function. Can we deprecate it?

Reply:
I've backed out this change, and will follow-up in a separate issue/PR later.

Reply:
xref #19811